woocommerce/woocommerce-ios
# Shipping Labels: display and pass the package details info into the creation form

Closes #4154 

## Description
In this PR I addressed all the data sent back and forth between the Shipping Label Creation Form and the Package Details flow.
I also display all the data in the Creation Form about the selected Package.

## Testing
1. Launch the app in debug mode (because of the feature flag).
2. Open an order detail.
3. Press the button "Create Shipping Label".
4. Press the button "Continue" in Create Shipping Label form, under the **Ship From** cell. Confirm the address.
5. Press the button "Continue" in Create Shipping Label form, under the **Ship To** cell. Confirm the address.
6. Press the button "Continue" in Create Shipping Label form, under the **Package Details** cell.
7. You should see the new screen that shows the Package Details screen and all the items that will be shipped.
8. Press the "Package selected" row. 
9. Select a package, then press "Done".
10. Add a Total package weight, press "Done".
11. You should see the selected data inside the Package Details cell, and the "Shipping Carrier and Rates" cell should be enabled now.
12. Re-open the Package Details screen. The selected information (package and total package weight) should be populated with the existing data.

## GIF
<img src="https://user-images.githubusercontent.com/495617/117984054-c8e5ab00-b337-11eb-8ca2-7a3d2f79bef5.gif" width=300 />

Update release notes:
